app/com/arpnetworking/metrics/portal/alerts/impl/PluggableAlertRepository.java

```diff
@@ -214,7 +214,11 @@ private void reload(final InputStream stream) {
         try {
             group = _objectMapper.readValue(bufferedStream, AlertGroup.class);
         } catch (final IOException e) {
-            throw new RuntimeException("Could not load alerts", e);
+            LOGGER.error()
```

Comment:
What is the motivation behind this change? Wouldn't you want the failure to be more noticeable if you cant read the alerts?

Reply:
This is to decrease the granularity of failure. If we push a bad config, we'll still become aware of it through other means e.g. we have an alert on time since most recent evaluation that would fire.

The issue I want to avoid is a bad alert config causing metrics portal to fail to start up entirely (which is what happened in this case). That prevents other unrelated jobs from running such as rollups and proxied metrics requests.

Reply:
Sounds good. As long as there's some way that we're notified of a bad config

Reply:
Can we log a metric here too?